package.json

```
@@ -49,10 +43,14 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/kentcdodds/match-sorter.git"
+    "url": "https://github.com/kentcdodds/match-sorter"
   },
   "bugs": {
     "url": "https://github.com/kentcdodds/match-sorter/issues"
   },
-  "homepage": "https://github.com/kentcdodds/match-sorter#readme"
+  "homepage": "https://github.com/kentcdodds/match-sorter#readme",
+  "engines": {
+    "node": ">=10",
+    "npm": ">=6"
+  }
```

Engines aren't necessary because this is client-side only.

```suggestion
  "homepage": "https://github.com/kentcdodds/match-sorter#readme"
```

---

It's necessary for `kcd-scripts`' compilation step, otherwise it will use `10.13` by default.

---

Not for client side. When using the `--bundle` flash, it uses roll-up